composer.json

```diff
@@ -29,13 +29,19 @@
     "autoload": {
         "psr-4": { "Omnipay\\Mollie\\" : "src/" }
     },
+    "autoload-dev": {
+        "psr-4": { "Omnipay\\Mollie\\Test\\": "tests/" }
+    },
     "require": {
         "omnipay/common": "^3"
     },
     "require-dev": {
         "omnipay/tests": "^3",
```

Comment:
Can you bump this to ^3.1, that should fix the lowest dependencies. (I've updated the php-mock-client dependency in 3.1)

Reply:
(referring to omnipay/tests)